### DISCUSSION TITLE
standup--2021.06.09

### DISCUSSION THREAD

**Discussion by amgando** [near-in-minutes/community#26] [Meetings]
attendees:
- @norrec99 
- @ozanisgor 
- @hiba-machfej
- @BigDaddy-Zephyr 
- @amgando 

topic
daily standup to sync on NEAR in Minutes development

**Comment by amgando** [comment]
### 💡  idea


capture discussions URL per video so we can follow up with discussions for each video.  these can be QA type discussions or general comments

**Comment by amgando** [comment]
### process

1. user submits new video (request, translation, content, etc) via Github issues
2. issue is closed as soon as the content is created / accepted and submitted to Airtable
3. any follow up happens via dedicated discussions per video

**Comment by ozanisgor** [comment]
**topic suggestion**

📌 How to do update on Airtable? For example, if I want to update video link or the description can I simply come to `translations` tab and in the `all` section and double click to what I want to update and change the text over there?
